src/facet/simulation/_simulation.py

```diff
@@ -83,7 +83,7 @@ class UnivariateSimulationResult(Generic[T_Partition]):
     feature_name: str
 
     #: Name of the target for which outputs are simulated.
-    output_name: str
+    output_name: Union[str, Sequence[str]]
```

Comment:
How can the output name be a sequence?

Reply:
It's because of the support for multioutput target. That's the type of the argument that is being passed to this constructor: `output_name: Union[str, Sequence[str]]` in `__init__`. My approach for enabling mypy was to focus on correcting type annotations instead of changing business logic to avoid braking it. I think there are two solutions:
1. Rename `output_name` to something that will also mean possibility of multiple output targets 
2. Change logic to disable multioutput and then only single output name can be passed

Which one should we go?

Reply:
Thanks. There is also logic built into the simulator classes to ensure we're only simulating single-target learners. So we should find the appropriate place to cast to `str`

Reply:
Sure, I found it - I added proper cast.